app/controllers/slices_controller.rb

```diff
@@ -9,7 +9,7 @@ class SlicesController < ActionController::Base
   append_view_path(File.join(Rails.root, *%w[app slices]))
 
   around_filter :set_locale
-  define_callbacks :render_page, terminator: "response_body"
+  define_callbacks :render_page
```

Comment:
Ok they were deprecated. But why does that mean this can be _removed_? (I have never figured out what this line does.)